lit_stablelm/model.py

```diff
@@ -63,6 +64,67 @@ def from_name(cls, name: str) -> Self:
     "pythia-6.9b": dict(block_size=2048, n_layer=32, n_embd=16384, n_head=32, padding_multiple=128),
     # https://huggingface.co/EleutherAI/pythia-12b/blob/main/config.json
     "pythia-12b": dict(block_size=2048, n_layer=36, n_embd=20480, n_head=40, padding_multiple=128),
+    # togethercomputer
```

Comment:
This dictionary has gotten quite long. I'll split this into a `configs.py` file in a separate PR